app/src/main/java/com/partymusicq/android/ui/fragments/PlayerFragment.kt

```diff
@@ -81,26 +83,60 @@ class PlayerFragment : Fragment(), UtilSpotify.SpotifyListener {
         }
     }
 
-    private fun setupOnClicks() {
-        prevButton.setOnClickListener(object : View.OnClickListener{
-            override fun onClick(p0: View?) {
-                TODO("not implemented") //To change body of created functions use File | Settings | File Templates.
-            }
-        })
-        playPauseButton.setOnClickListener(object : View.OnClickListener{
-            override fun onClick(p0: View?) {
-                if (spotifyAppRemote == null || spotifyAppRemote?.isConnected != true) {
-                    UtilSpotify.logIntoSpotify(SpotifyEvent(SpotifyEventEnum.LoginAndPlayPause))
-                } else {
-                    UtilSpotify.playPause()
-                }
+    private fun initFirebase() {
```

Comment:
The function does more that what the function name says it does. I think that the function needs to be renamed.

Reply:
yeah I didn't want to init spotify here but i have to do it in the callback. initFirebaseAndSpotfy is a better name